ext/drbbdup/drbbdup.h

```diff
@@ -257,8 +258,32 @@ typedef struct {
      * thread before it becomes a candidate for dynamic generation.
      */
     ushort hit_threshold;
+    /**
+     * Determines whether drbbdup should track a variety of statistics. Note keeping track
+     * of statistics incurs additional overhead and it is not recommended at deployment.
+     *
+     * Set to true if the client calls drbbdup_get_stats().
```

Comment:
This could confuse people: it could be read as "this is automatically set to true if the client ever calls drbbdup_get_stats".  Adding a few more words should clarify it.

Reply:
Done